src/main/java/com/synopsys/integration/alert/database/api/configuration/DescriptorAccessor.java

```diff
@@ -153,9 +153,9 @@ public boolean registerDescriptor(final String descriptorName, final DescriptorT
             final RegisteredDescriptorEntity newDescriptor = new RegisteredDescriptorEntity(descriptorName, descriptorTypeId);
             final RegisteredDescriptorEntity createdDescriptor = registeredDescriptorRepository.save(newDescriptor);
             addFieldsAndUpdateRelations(createdDescriptor.getId(), descriptorFields);
-            return true;
+            return createRegisteredDescriptorModel(createdDescriptor);
         }
-        return false;
+        return createRegisteredDescriptorModel(optionalDescriptorEntity.get());
```

Comment:
I know we do an `if (optionalDescriptorEntity.isEmpty())` check in the block above this, but I would feel much more comfortable doing an `Optional.get()` inside of an if statement (if the condition above ever changed, this would not necessarily be protected by it which could introduce a bug).